File: bookBeauty.Model/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace bookBeauty.Model
{
    public class Order
    {
        public int OrderId { get; set; }

        public float? TotalPrice { get; set; }

        public DateTime? DateTime { get; set; }

        public int CustomerId { get; set; }

        public string? OrderNumber { get; set; }

        public string Status { get; set; }

        public string? Address { get; set; }

        public virtual ICollection<OrderItem> OrderItems { get; set; } = new List<OrderItem>();


    }
}

File: UI/bookbeauty_mobile/lib/models/order.dart
import 'package:json_annotation/json_annotation.dart';

part 'order.g.dart';

@JsonSerializable()
class Order {
  int? orderId;
  double? totalPrice;
  DateTime? dateTime;
  int? customerId;
  String? orderNumber;
  String? status;
  String? address;

  Order({
    this.orderId,
    this.totalPrice,
    this.dateTime,
    this.customerId,
    this.orderNumber,
    this.status,
    this.address,
  });

  factory Order.fromJson(Map<String, dynamic> json) => _$OrderFromJson(json);

  Map<String, dynamic> toJson() => _$OrderToJson(this);
}
